zwave_js_server/model/controller.py

```diff
@@ -167,7 +167,7 @@ async def async_stop_exclusion(self) -> bool:
 
     async def async_remove_failed_node(self, node_id: int) -> None:
         """Send removeFailedNode command to Controller."""
-        await self.client.async_send_json_message(
+        await self.client.async_send_command(
```

Comment:
Should we rename `async_send_command` to just `send_command`?

Reply:
Yes, but let's do that another time. I didn't want to make a breaking change in this PR.